admin-ui/plugins/auth-server/plugin-metadata.js

```diff
@@ -176,6 +180,11 @@ const pluginMetadata = {
       path: PLUGIN_BASE_APTH + '/authn',
       permission: SCOPE_READ,
     },
+    {
+      component: LdapAddPage,
+      path: PLUGIN_BASE_APTH + '/authn/ldap/new',
+      permission: SCOPE_WRITE,
+    },
```

Comment:
Using a generic permission constant (`SCOPE_WRITE`) instead of a more specific one (`LDAP_WRITE`) is bad because it violates the Principle of Least Privilege by failing to use the most specific permission available, which could lead to unintended access if the generic permission is granted in a context where the specific one is not.